install.sh

```diff
@@ -1,4 +1,26 @@
-mkdir ~/.scripts
-cp ./git-extensions/* ~/.scripts 
-echo 'export PATH=$PATH:~/.scripts' >> ~/.zshrc 
-export PATH=$PATH:~/.scripts
+#!/bin/bash
+
+shell_config(){
+  # find the defualt shell
+  defualt_shell=`echo $SHELL | awk -F '/' '{ print $NF }'`
```

Comment:
### find the default shell with AWK
get last field from output of the 'echo $SHELL' command separated by '/'.